frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -34,7 +34,7 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
   React.useEffect(() => {
     k8sGet(ClusterServiceVersionModel, appName, ns)
       .then((clusterServiceVersionObj) => {
-        setIsIndependent(checkForIndependentSupport(clusterServiceVersionObj));
+        setIsIndependent(supportAnnotation(clusterServiceVersionObj).includes('external'));
```

Comment:
NIT: `getAnnotaion` is available in common.ts of console/shared. Try to define feature annotation in the constant file. Use it from there.

const featureAnnotations = 'features.ocs.openshift.io/enabled'
getAnnotations(clusterServiceVersionObj)[featureAnnotations].includes('external')

Reply:
@cloudbehl using function from console/shared are popping up error `k8s-models.ts:1 Uncaught ReferenceError: Cannot access 'allModels' before initialization` as features.ts from plugins are loaded before some of the modules. Also, we should keep the annotation name in one file only (features.ts) as if annotation changes, we can change it at one place only.

Reply:
> @cloudbehl using function from console/shared are popping up error `k8s-models.ts:1 Uncaught ReferenceError: Cannot access 'allModels' before initialization` as features.ts from plugins are loaded before some of the modules. 
Bit weird that you are not able to use the function. 


> Also, we should keep the annotation name in one file only (features.ts) as if annotation changes, we can change it at one place only.

Exactly my point we need to have commonplace for annotations.

@bipuladh ^^

Reply:
ack